src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.scss

```diff
@@ -0,0 +1,99 @@
+@import '../../../common/variables';
+@import './mixins';
+
+.bcs-AvatarGroup {
+    display: flex;
+    margin: 10px 0;
+
+    .bcs-AvatarGroup-avatar {
+        height: 32px;
+        width: 32px;
+    }
+
+    .bcs-AvatarGroup-overflowCount {
+        align-items: center;
+        background: $bdl-neutral-02;
+        border-radius: 100%;
+        color: #fff;
+        cursor: pointer;
+        display: flex;
+        font-weight: bold;
+        justify-content: center;
+    }
+
+    $avatar-spacing: 4px;
+
+    .bcs-AvatarGroup-avatarContainer + .bcs-AvatarGroup-avatarContainer {
+        margin-left: $avatar-spacing;
+    }
+
+    .bcs-AvatarGroup-overflowCountContainer {
+        margin-left: $avatar-spacing;
+
+        &:hover,
+        &:active {
+            margin-left: $avatar-spacing;
+        }
+    }
+}
+
+.bcs-AvatarGroup-listFlyout {
+    max-width: 100%;
+    width: 280px;
+
+    .bcs-AvatarGroup-list {
```

Comment:
The renaming done so far is a great start. I think we should target these components for further improvement in the future, as well. It looks like this should probably be its own component (`bcs-AvatarList` or something similar), for example, which could then have `bcs-AvatarList-title`, `bcs-AvatarList-item`, etc.

Reply:
Yep agreed. Since this isn't yet a fully generic reusable component separate from tasks (yet), I didn't go all the way with that, but if/when we extract it fully from the tasks stuff we can fix that.